### PR TITLE
sticker api

### LINKED ISSUE #32: Implement discords sticker API
## As a user I want to...
be able to call the [sticker](https://discord.com/developers/docs/resources/sticker) endpoints

## So that...
I can perform sticker actions using my discord bot

## Context
Discord provides a rich set of APIs for interacting with channels in a discord server. We should implement all these endpoints. This ticket covers only the sticker endpoints. https://discord.com/developers/docs/resources/sticker

## Acceptance Criteria
GIVEN a user wants to send a request to interact with a sticker on discord
THEN the specified action should be executed with sufficent error logging in place

### PR DESCRIPTION
**Pull request by nateweisz**
Closes #32

### CODE REVIEW

**Comment by nateweisz** [comment]
**PR on hold until #62 is merged!**

**Review by surajkumar — APPROVED** [review]

**Pull request merged**
